erpnext/setup/doctype/company/company.json

```diff
@@ -728,35 +737,98 @@
    "options": "Terms and Conditions"
   },
   {
+   "depends_on": "eval:!doc.__islocal",
    "fieldname": "default_use_tax_expense_account",
    "fieldtype": "Link",
    "label": "Default Use Tax Expense Account",
    "options": "Account"
   },
   {
+   "depends_on": "eval:!doc.__islocal",
```

Comment:
what is the purpose of this condition?

Reply:
- When creating a new company, the Accounts are not created.
- So making the Accounts field will not have the appropriate Link Field values.
- Corresponding Accounts for a company are created while inserting.
- what depends on does is, it'll hide the field if we are creating a new company and make the field visible after saving